proto/internal/uber/cadence/matching/v1/service.proto

```diff
@@ -291,3 +293,8 @@ message RefreshTaskListPartitionConfigRequest {
 message RefreshTaskListPartitionConfigResponse {
 
 }
+
+message AutoConfigHint {
```

Comment:
I think we can just reuse the one defined in https://github.com/cadence-workflow/cadence-idl/pull/192/files

Reply:
TaskListPartitionConfig was duplicated because I defined the one inside matching first and then I released that I need to expose it to frontend

Reply:
Yeah, fixed